build/notice-file/Readme.md

```diff
@@ -0,0 +1,45 @@
+# Notice.txt File Configuration
+
+We are automatically generating Notice.txt by using first-level dependencies of the project. The related pipeline uses `config.yaml` stored in this folder.
+
+
+## Configuration
+
+Notice.txt will be always generated from first level dependencies scanned from package.json files. For some cases it may be useful to define 
```

Comment:
```suggestion
Notice.txt will be always generated from first-level dependencies scanned from package.json files. For some cases it may be useful to define the
```